fdb-relational-core/src/main/java/com/apple/foundationdb/relational/recordlayer/query/PlanGenerator.java

```diff
@@ -190,7 +190,7 @@ private Plan<?> getPlanInternal(@Nonnull String query, @Nonnull KeyValueLogMessa
                                     final var result = (QueryPlan.PhysicalQueryPlan) candidate;
                                     final var candidateQueryPlan = result.getRecordQueryPlan();
                                     var bestQueryPlan = acc == null ? null : ((QueryPlan.PhysicalQueryPlan) acc).getRecordQueryPlan();
-                                    if (bestQueryPlan == null || new CascadesCostModel(planner.getConfiguration()).compare(candidateQueryPlan, bestQueryPlan) < 0) {
+                                    if (bestQueryPlan == null || new PlanningCostModel(planner.getConfiguration()).compare(candidateQueryPlan, bestQueryPlan) < 0) {
```

Comment:
@hatyo This is probably not a great as the cost model might not be the quickest thing in the world.

Reply:
I see and I think I remember now. Just stumbled over this and had no memory of try of this :-)